src/ui/public/chrome/directives/global_nav/global_nav.less

```diff
@@ -33,7 +33,7 @@
     width: @global-nav-open-width;
     list-style-type: none;
     &.kibana {
-      background-image: url("~ui/images/grimoire-dashboard.svg");
+      background-image: url("~ui/images/bitergia-analytics.svg");
```

Comment:
This image file must be submitted. :thinking:

Reply:
Yes, my fault, adding the image ... done! @sanacl could you recheck? Thanks!